Source/NSStoryboardSegue.m

```diff
@@ -143,6 +171,56 @@ - (void) perform
           RETAIN(w);
         }
     }
+  else if ([_kind isEqualToString: @"popover"])
+    {
+      NSPopover *po = [[NSPopover alloc] init];
+      NSRect rect = [_popoverAnchorView frame];
+      NSRectEdge edge = NSMinXEdge;
+      NSPopoverBehavior behavior = NSPopoverBehaviorApplicationDefined; 
+      
+      // Convert edge...
+      if ([_preferredEdge isEqualToString: @"maxY"])
```

Comment:
Maybe it would be better to already decode these values as the enum values?

Reply:
Yes, I am going to work on that.